### ISSUE TITLE
Strange swagger rendering when using @ResponseHeaders

### ISSUE THREAD

**Issue by vguna** [stoicflame/enunciate#398]
Hi.

I'm using enunciate 2.3.0 and trying to document an endpoint using `@ResponseHeader` and `@HTTP` like this:

```java
	 * @ResponseHeader Location The location of the created login.
	 *
	 * @HTTP 201 if creation was successful.
	 * @HTTP 400 if there are errors during validation.
```

Looking at the rendered swagger output, leads to the status code listing for 400, assigning the response header definition to it as well. Although it should be separated like in the normal enunciate docs.
Normally I would describe in the `@ResponseHeader`, that it is only set on 201. But now it looks like it's also returned on 400. Also the model is added there as well - which I don't understand :).

Better see the picture attached for a better understanding.

![screen shot 2016-05-19 at 12 00 47](https://cloud.githubusercontent.com/assets/3257311/15390226/129eed96-1dbb-11e6-9b25-462652db2be9.png)

**Comment by stoicflame** [comment]
Fixed at 7f9f69f.

**Comment by stoicflame** [comment]
[Enunciate 2.4 has been released](https://github.com/stoicflame/enunciate/releases/tag/v2.4.0).
